layer-atlas/src/main/java/com/layer/atlas/adapters/AtlasConversationsAdapter.java

```diff
@@ -242,6 +262,91 @@ public void onBindViewHolder(ViewHolder viewHolder, int position) {
         }
     }
 
+    private void sortAvatars(final Set<Identity> participants,
+                             final Conversation conversation,
+                             final ViewHolder viewHolder) {
+        compositeDisposable.add(Observable.fromIterable(participants)
+                .map(new Function<Identity, String>() {
```

Comment:
why not use lambdas?

Reply:
I don't want to add Java 8 support because I would like to add only minimum dependencies.

Reply:
Yeah, I understand, but on the other hand it could improve readability very very much :)